rpcserverhandlers_test.go

```diff
@@ -3731,6 +3748,44 @@ func TestHandleSubmitBlock(t *testing.T) {
 	}})
 }
 
+func TestHandleVerifyChain(t *testing.T) {
+	t.Parallel()
+
+	hash := mustParseHash("00000000000000000c07ff0178ad600db22c18f8f47fa423fa144b2ca919475d")
+	depth := int64(50)
+	level := int64(1)
+
+	testRPCServerHandler(t, []rpcTest{{
+		name:    "handleVerifyChain: ok",
+		handler: handleVerifyChain,
+		cmd:     &types.VerifyChainCmd{},
+		result:  true,
+	}, {
+		name:    "handleVerifyChain: ok with checklevel=1",
+		handler: handleVerifyChain,
+		cmd: &types.VerifyChainCmd{
+			CheckLevel: &level,
+		},
+		result: true,
+	}, {
+		name:    "handleVerifyChain: no block exists",
+		handler: handleVerifyChain,
+		cmd: &types.VerifyChainCmd{
+			CheckDepth: &depth,
+		},
+		mockChain: func() *testRPCChain {
+			chain := defaultMockRPCChain()
+			chain.bestSnapshot = &blockchain.BestState{
+				Height: 100,
+				Hash:   *hash,
+			}
+			chain.blockByHeightErr = errors.New("no block exists")
+			return chain
+		}(),
+		result: false,
+	}})
```

Comment:
Looks like this is also not covering these branches in the `verifyChain` helper that this handler relies:
- https://github.com/decred/dcrd/blob/66807231ce051a898eb4ec8ff86986d8c4d47959/rpcserver.go#L4747-L4749
- https://github.com/decred/dcrd/blob/66807231ce051a898eb4ec8ff86986d8c4d47959/rpcserver.go#L4766-L4771